1.0.2/build-msbuild/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# TODO pull from microsoft/dotnet when these become available
+FROM microsoft/dotnet-nightly:1.0.2-sdk-msbuild
```

Comment:
That was a mis-tagged image and deleting tags is currently broken in DH.  You should be referring to the readme for the officially supported images.